### DISCUSSION TITLE
How to export a  A256CBC-HS512 key and import it as a Buffer/crypto KeyObject?

### DISCUSSION THREAD

**Discussion by BioSurienDG** [panva/jose#573] [Q&A] [answered]
Currently what I've been doing is just `Buffer.from(await jose.exportJWK(await jose.generateSecret('A256CBC-HS512')).k)`. However, the length of the buffer is always 688 bits instead of 512 bits. Am I doing something wrong here?

**Comment by panva** [comment, marked as answer]
k in jwk is base64url encoded

**Reply by BioSurienDG** [reply]
Thank you 🙂
